### DISCUSSION TITLE
Contribuições no Currículo

### DISCUSSION THREAD

**Discussion by mthalesb** [frontendbr/forum#1740] [Discussão]
Bom dia. Tinha visto faz tempo um vídeo falando que os programadores devem colocar no currículo as contribuições que fizeram para algum projeto . Logo, minha pergunta é, qual o melhor lugar para adicionar tal informação? No campo das experiências?

**Comment by giovannism20** [comment]
Normalmente vejo pessoas colocando na área de Experiência como você citou.

**Comment by LuanDc** [comment]
Coloca os projetos mais relevantes na área de experiência e o link pro seu github. Na entrevista você pode explicar melhor sobre suas contribuições na comunidade, desenvolvimento de projetos pessoais e experiências profissionais.

**Comment by ninetails** [comment]
Se está falando de **LinkedIn**, há uma seção **Conquistas**. Lá vc pode colocar projetos e no que vc contribuiu, adicionando também seus **coworkers** que participaram ativamente do projeto.
